src/elements/http-transaction.js

```diff
@@ -9,15 +9,25 @@ export default function (namespace) {
     }
 
     get request() {
-      return this.children(item => item.element === 'httpRequest').first();
+      return this.children.filter(item => item.element === 'httpRequest').first();
     }
 
     get response() {
-      return this.children(item => item.element === 'httpResponse').first();
+      return this.children.filter(item => item.element === 'httpResponse').first();
     }
 
     get authSchemes() {
-      return namespace.toElement(this.attributes.get('authSchemes').map(item => (new AuthScheme()).fromRefract(item)));
```

Comment:
Not too happy about what the `authSchemes` function did previously and continues to do here. At least this change no longer uses re-serialisation to/from JSON.